.gitignore

```diff
@@ -45,3 +45,6 @@ node_modules
 
 # Uploads
 /public/uploads
+
+# Storybook
+/public/design
```

Comment:
With this here, that means all the committed files below that are from /public/design should go away